### PR TITLE
WPF 890011 - Images added for Scheduler tooltip and cell padding support

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have added images and GIF files for the Scheduler tooltip and cell padding support.

### CODE REVIEW

File: wpf/Scheduler/Appointments.md [hunk 1]
```diff
@@ -774,6 +774,8 @@ this.Schedule.EnableToolTip = true;
 {% endhighlight %}
 {% endtabs %}
 
+![Appointment tooltip in WPF Scheduler](appointments_images/Appointment-tooltip.gif)
```
Comment (Karthickmani97) [inline wpf/Scheduler/Appointments.md:777]: Change alt text to `appointment-tool-tip-support-in-WPF-scheduler`

Image name to `appointment-tool-tip-support-in-WPF-scheduler`

File: wpf/Scheduler/DaysView.md [hunk 2]
```diff
@@ -456,6 +456,8 @@ this.Schedule.DaysViewSettings.CellRightPadding = 30;
 {% endhighlight %}
 {% endtabs %}
 
+![Cell Right Padding in Scheduler Appointment](DaysView_Images/wpf-cell-padding.png)
```
Comment (Karthickmani97) [inline wpf/Scheduler/DaysView.md:459]: Change alt text to `cell-right-padding-support-in-WPF-scheduler`

Image name to `cell-right-padding-support-in-WPF-scheduler`

Change alt text and image name in all places.

File: wpf/Scheduler/Month-View.md [hunk 3]
```diff
@@ -562,6 +562,8 @@ this.Schedule.MonthViewSettings.CellRightPadding = 30;
 {% endhighlight %}
 {% endtabs %}
 
+![Cell Right Padding in Scheduler Appointment](Month-View_Images/wpf-cell-padding.png)
```
Comment (Karthickmani97) [inline wpf/Scheduler/Month-View.md:565]: Having gap between image and border.

Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/Month-View.md]: Updated.

File: wpf/Scheduler/Appointments.md [hunk 4]
```diff
@@ -774,6 +774,8 @@ this.Schedule.EnableToolTip = true;
 {% endhighlight %}
 {% endtabs %}
 
+![appointment-tool-tip-support-in-WPF-scheduler](appointments_images/Appointment-tooltip.gif)
```
Comment (Karthickmani97) [inline wpf/Scheduler/Appointments.md:777]: Change image name to `appointment-tool-tip-support-in-WPF-scheduler`

File: wpf/Scheduler/DaysView.md [hunk 5]
```diff
@@ -456,6 +456,8 @@ this.Schedule.DaysViewSettings.CellRightPadding = 30;
 {% endhighlight %}
 {% endtabs %}
 
+![cell-right-padding-support-in-WPF-scheduler](DaysView_Images/wpf-cell-padding.png)
```
Comment (Karthickmani97) [inline wpf/Scheduler/DaysView.md:459]: Change image name to `cell-right-padding-support-in-WPF-scheduler`

change in all places.

File: wpf/Scheduler/TimelineView.md [hunk 6]
```diff
@@ -467,6 +467,8 @@ this.Schedule.TimelineViewSettings.CellBottomPadding = 30;
 {% endhighlight %}
 {% endtabs %}
 
+![cell-bottom-padding-support-in-WPF-scheduler](Timeline_images/wpf-cell-padding.png)
```
Comment (Karthickmani97) [inline wpf/Scheduler/TimelineView.md:470]: timeline view image - Not properly update based on code snippet added.

**Review by Karthickmani97 — COMMENTED** [review]
Share review correction changes in new PR.

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

File: wpf/Scheduler/Appointments.md (hunk 4 above)
Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/Appointments.md]: Changed

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

File: wpf/Scheduler/TimelineView.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/TimelineView.md]: Changed.

**Pull request merged**